PRFCS/prfc-1.md

```diff
@@ -108,7 +112,7 @@ In this section, `++` denotes bytes concatenation.
 | Field | Value |
 | ----- | ----- |
 | Topic | `0u8 ++ owner_address ++ recipient_address: Option<PublicAddress>`  |
```

Comment:
Should we still specify the type for owner_address too? just like how you did in PRFC2. Same applies for SetAllowance